src/homework4/task7.py

```diff
@@ -0,0 +1,16 @@
+"""Homework 4 - Task7
+
+Оглянемся назад. Числа
+Даны два натуральных числа. Вычислите их наибольший общий делитель
+при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+a = 248
+b = 12
+
+while a and b:
+    if a > b:
+        a %= b
+    else:
+        b %= a
+print(a + b)
```

Comment:
it's a cheat. you don't know where is solution (the second is 0). that's mean you haven't implemented the euclidean algorithm properly